tests/pytest/core/test_admin.py

```diff
@@ -75,3 +75,38 @@ def test_pre_login_user_add_staff_to_group(mocker, model_AdminUser):
 
     staff_group = Group.objects.get(name=STAFF_GROUP_NAME)
     assert model_AdminUser.groups.contains(staff_group)
+
+
+@pytest.mark.django_db
+def test_pre_login_user_does_not_add_transit_staff_to_group(mocker, model_AdminUser):
+    mocked_request = mocker.Mock()
+    mocked_request.session.get.return_value = None
+
+    mocker.patch.object(benefits.core.admin.settings, "GOOGLE_SSO_STAFF_LIST", [model_AdminUser.email])
```

Comment:
To override settings for a test, we want to use the `settings` fixture provided by `pytest-django` (already installed): https://pytest-django.readthedocs.io/en/latest/helpers.html#settings

See e.g. https://github.com/cal-itp/benefits/blob/main/tests/pytest/test_secrets.py#L80

Reply:
Ah good catch. I think this means `test_pre_login_user_add_staff_to_group`, which existed before this PR, should do the same

Reply:
Created https://github.com/cal-itp/benefits/issues/2316 for the test unrelated to this PR